src/data/complex/exponential_bounds.lean

```diff
@@ -31,24 +33,48 @@ begin
   rw [_root_.abs_one, abs_of_pos]; norm_num1,
 end
 
-lemma exp_one_gt_271828182 : 2.71828182 < exp 1 :=
+lemma exp_one_gt_271828182 : 2.7182818283 < exp 1 :=
```

Comment:
These lemma names should be updated.

Reply:
I think the lemma names should be a little less explicit about the number, perhaps `exp_one_gt_d10` for "to 10 decimals". I don't think I will be able to remember or reproduce the name otherwise....

Reply:
Amended, thanks both.